### PR TITLE
Jmodal+Jsmodal1 (Code Review)

### PR DESCRIPTION
**Pull request by AmineHLub**
<h1>Summary</h1>

- Create JavaScript file "dynamic-html.js" (Created Arr of obj containing data, data will be passed to html and modals, created a function to generate modals on click depending on data,button clicked and the arr of objs, and finally created multiple dynamic styles.)
- Created Styling for both Desktop and mobile version.

<h2>PS : </h2> This project was supposed to be on Jsmodal only but When I showed my partner how to change a branch's name using my own branch I forgot to Revert changes so the commits start from Jsmodal > Jsmodal2

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dynamic-html.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
+// section Created
+
+const arr = [
+  {
+    close: './project-images/Icon.png',
+    logo: './project-images/one.png',
+    title: 'Random name for project 1',
+    tech: ['Ruby', 'css', 'JavaScript', 'html'],
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum recusandae accusamus minima et officiis vel, unde cum saepe officia, iusto eaque doloremque commodi quisquam rem nisi error illum ipsa repellat!',
```
Comment (mikenath223) [inline dynamic-html.js]: - [ ] On the modal pop-up kindly add padding to the project description.

File: index.html [hunk 2]
```diff
@@ -73,88 +74,65 @@ <h1 class="recent">My Recent Works</h1>
...
-            <ul class="languages">
-                <li><button class="lang-btns">Ruby on rails</button></li>
-                <li><button class="lang-btns">css</button></li>
-                <li><button class="lang-btns">JavaScript</button></li>
-                <li><button class="lang-btns">html</button></li>
+            <ul class="languages customlng4">
                 </ul>
         </section>
-        <section class="flexible-items-inside-grid"><button class="prjct-btn">See Project</button></section>
+        <section class="flexible-items-inside-grid"><button onclick="modalCreation(3)" class="prjct-btn">See Project</button></section>
```
Comment (mikenath223) [inline index.html]: - [ ] Please use the dom selector query to add dom onclick events.

Comment (mikenath223) [reply index.html]: - [ ] Kindly make the change on all `See Project` buttons.

File: dynamic-html.js (hunk 1 above)
Comment (AmineHLub) [reply dynamic-html.js]: I fixed that issue thank you very much

File: index.html (hunk 2 above)
Comment (AmineHLub) [reply index.html]: Done, Thank you for your code review :+1:

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @AmineHLub, 

### You can consider:

- [ ] - Create "css" folder and put all .css files inside this folder.
- [ ] - Create "js" folder and put all .js files inside this folder.
- [ ] - Create "img" folder and put all .png, .jpeg, .jpg, .svg files (all images and vectors) inside this folder.

It can help you to maintain your repository more ordered and clear 😉 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**